src/Azs.ContainerRegistry.Admin/readme.md

```diff
@@ -75,13 +75,53 @@ directive:
     set:
       subject-prefix: ContainerRegistry
 
+    # Rename cmdlet parameter name and set default value in ContainerRegistryQuota
+  - where:
+      verb: New
+      subject: Quota
+      parameter-name: capacityPerRegistryInGiB
+    set:
+      default:
+        script: '100'
+  - where:
+      subject: Quota
+      parameter-name: NumberOfRegistry
+    set:
+      parameter-name: NumberOfRegistries
+  - where:
+      verb: New
+      subject: Quota
+      parameter-name: NumberOfRegistries
+    set:
+      default:
+        script: '20'
+		
     # Rename cmdlet parameter name in ContainerRegistrySetup
   - where:
       subject: ContainerRegistrySetup
       parameter-name: SslCertBase64
     set:
       parameter-name: SslCert
+
+    # Rename model property names
+  - where:
```

Comment:
Did you check the output model to make sure it is renamed there as well?

Reply:
@bganapa did you mean this?
PS C:\Windows\system32> Get-AzsContainerRegistryQuota -Name "Default quota" | ConvertTo-Json
{
    "CapacityPerRegistryInGiB":  100,
    "Id":  "/subscriptions/5d5412b9-9a29-4c8a-acd7-b3955625860e/providers/Microsoft.ContainerRegistry.Admin/locations/redmond/quotas/Default quota",
    "Name":  "redmond/Default quota",
    **"NumberOfRegistries"**:  3,
    "Type":  "Microsoft.ContainerRegistry.Admin/locations/quotas"
}